web-server/v0.4/src/pages/Explore/index.js

```diff
@@ -85,16 +96,38 @@ class Explore extends Component {
         title: 'Action',
         dataIndex: '',
         key: 'action',
-        render: (text, record) => (
-          <Popconfirm
-            title="Start a new dashboard session?"
-            cancelText="No"
-            okText="Yes"
-            onConfirm={() => this.startSharedSession(record)}
-          >
-            <Button type="link">Start Session</Button>
-          </Popconfirm>
-        ),
+        children: [
+          {
+            title: 'Start Session',
+            dataIndex: '',
+            key: 'start',
+            render: (text, record) => (
+              <Popconfirm
+                title="Start a new dashboard session?"
+                cancelText="No"
+                okText="Yes"
+                onConfirm={() => this.startSharedSession(record)}
+              >
+                <Button type="link">Start Session</Button>
+              </Popconfirm>
+            ),
+          },
+          {
+            title: 'Delete Session',
+            dataIndex: '',
+            key: 'start',
+            render: (text, record) => (
+              <Popconfirm
+                title="Delete the dashboard session?"
+                cancelText="No"
+                okText="Yes"
+                onConfirm={() => this.deleteSharedSession(record)}
+              >
+                <Icon type="delete" theme="twoTone" style={{ textAlign: 'right' }} />
```

Comment:
What are your thoughts on making the `Delete Session` action a separate column as opposed to a nested column with `<Button type="link">Delete Session</Button>` as the rendered element? This approach would help us avoid rendering an additional column header within the view.